scripts/buildTypes.mts

```diff
@@ -14,22 +14,32 @@ async function emitDeclarations(tsconfig: string, outDir: string) {
   await $$`tsc -p ${tsconfig} --outDir ${outDir} --declaration --emitDeclarationOnly`;
 }
 
-async function addImportExtensions(folder: string) {
+async function addImportExtensions(folder: string, removeCss: boolean) {
   // eslint-disable-next-line no-console
   console.log(`Adding import extensions`);
   const dtsFiles = await glob('**/*.d.ts', { absolute: true, cwd: folder });
   if (dtsFiles.length === 0) {
     throw new Error(`Unable to find declaration files in '${folder}'`);
   }
 
+  const babelPlugins: babel.PluginItem[] = [
+    ['@babel/plugin-syntax-typescript', { dts: true }],
+    ['@mui/internal-babel-plugin-resolve-imports'],
+  ];
+  if (removeCss) {
```

Comment:
So that you only have to install the package if you are using the feature which will be limited to one repo right now.

Reply:
we will bundle it with the script when we create a package for this

Reply:
Yeah. But that's when we move these to separate package.